src/agent/pnp_helper/inc/pnp_protocol.h

```diff
@@ -43,7 +43,7 @@ typedef void (*PnP_PropertyCallbackFunction)(
 // a result code or version.  It is used when sending properties that are NOT marked as <"writable": true> in the DTDL defining
 // the given property.
 //
-// The application itself needs to send this to Device Twin, using a function such as IoTHubDeviceClient_LL_SendReportedState.
+// The application itself needs to send this to Device Twin, using a function such as ClientHandle_SendReportedState.
 //
 STRING_HANDLE
 PnP_CreateReportedProperty(const char* componentName, const char* propertyName, const char* propertyValue);
```

Comment:
>PnP_CreateReportedProperty [](start = 0, length = 26)

where are the methods in this header defined now?

Reply:
There is a corresponding .c for this one. I just deleted the others. 

---
In reply to: [591975866](https://github.com/Azure/iot-hub-device-update/pull/12#discussion_r591975866) [](ancestors = 591975866)